pep-0664.rst

```diff
@@ -0,0 +1,98 @@
+PEP: 664
+Title: Python 3.11 Release Schedule
+Version: $Revision$
+Last-Modified: $Date$
+Author: Pablo Galindo Salgado <pablogsal@python.org>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 12-Jul-2021
+Python-Version: 3.11
+
+
+Abstract
+========
+
+This document describes the development and release schedule for
+Python 3.11.  The schedule primarily concerns itself with PEP-sized
+items.
+
+.. Small features may be added up to the first beta
+   release.  Bugs may be fixed until the final release,
+   which is planned for end of October 2021.
+
+Release Manager and Crew
+========================
+
+- 3.11 Release Manager: Pablo Galindo Salgado
+- Windows installers: Steve Dower
+- Mac installers: Ned Deily
+- Documentation: Julien Palard
+
+
+Release Schedule
+================
+
+3.11.0 schedule
+---------------
+
+Note: the dates below use a 17-month development period that results
+in a 12-month release cadence between major versions, as defined by
+PEP 602.
+
+Actual:
+
+- 3.11 development begins: Monday, 2021-05-03
+
+Expected:
+- 3.11.0 alpha 1: Monday, 2021-10-04
```

Comment:
Add a newline here for this to be rendered as a bullet list, at least in the GitHub preview:
```suggestion

- 3.11.0 alpha 1: Monday, 2021-10-04
```

https://github.com/python/peps/blob/1189e291b8985e78eae330dc4d9bbc116b2eb8bf/pep-0664.rst